pkg/agent/openflow/pipeline.go

```diff
@@ -269,11 +256,8 @@ var (
 	// ofPortMarkRange takes the 16th bit of register marksReg to indicate if the ofPort number of an interface
 	// is found or not. Its value is 0x1 if yes.
 	ofPortMarkRange = binding.Range{16, 16}
-	// ofPortRegRange takes a 32-bit range of register PortCacheReg to cache the ofPort number of the interface.
+	// ofPortRegRange takes a 31-bit range of register PortCacheReg to cache the ofPort number of the interface.
```

Comment:
Isn't the range inclusive so that 32-bit is correct?

Reply:
I must have changed it by accidentally. Let me revert it.